Makefile

```diff
@@ -82,7 +82,7 @@ cover: unit
 	go tool cover -html=coverage.out
 
 e2e:
-	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]" ./hack/e2e.sh
+	FOCUS="\[AzureClusterReader\]|\[CustomerAdmin\]|\[EndUser\]\[Fake\]" ./hack/e2e.sh
```

Comment:
the `[Real]` or `[Fake]` tags are only needed by tests interacting with an azure client. This change is not needed

Reply:
The azureclientreader test is using an azure client now.